### PR TITLE
documentation for the Minut Point

### PR DESCRIPTION
**Pull request by fredrike**
**Description:**

Added documentation for the Minut Point integration.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant):** home-assistant/home-assistant#17466

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/point.markdown [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+logo: minut.svg
+ha_category: Hub
+featured: true
+ha_iot_class: "Cloud Polling"
+---
+
+The Point component is the main component to integrate the [Minut Point](https://minut.se/). To connect Point, you will have to [sign up for a developer account](https://minut.com/community/developers/) and get a `client_id` and `client_secret` using the `callback url` shown in the integration flow.
+
+<p class='note'>
+The Point is just active occasionaly so the sensors are only updated every hour or so.
```
Comment (ludeeus) [inline source/_components/point.markdown]:
```suggestion
The Point is just active occasionally so the sensors are only updated every hour or so.
```

File: source/_components/point.markdown [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+The events sent from the Point is sent as a webhook back to Home Assistant with `event_type` as `point_webhook_received`, please consider the documentation for the [IFTT](https://www.home-assistant.io/components/ifttt/) component on how to write automations for webhooks.
+</p>
+
+### {% linkable_title Configuration %}
+
+Integration to Point is configured via the Integrations setting found in the Configuration section on your Home Assitant page.
+Just follow the integration dialogs and make sure that your Home Assistant instance is accessable from the web (to be able to recieve webhooks).
```
Comment (ludeeus) [inline source/_components/point.markdown]:
```suggestion
Just follow the integration dialogs and make sure that your Home Assistant instance is accessible from the web (to be able to receive webhooks).
```

File: source/_components/binary_sensor.point.markdown [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: "Minut Point Binary Sensor"
+description: "Access your Minut Point Events as binary senros."
```
Comment (ludeeus) [inline source/_components/binary_sensor.point.markdown]:
```suggestion
description: "Access your Minut Point Events as binary sensors."
```

File: source/_components/binary_sensor.point.markdown [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: minut.svg
+ha_category: BinarySensor
+ha_iot_class: "Cloud Push"
+---
+
+Each Point exposes the following Sensors:
```
Comment (ludeeus) [inline source/_components/binary_sensor.point.markdown]:
```suggestion
Each Point exposes the following binary sensors:
```
Unsure about this one(?)

File: source/_components/binary_sensor.point.markdown [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+layout: page
+title: "Minut Point Binary Sensor"
+description: "Access your Minut Point Events as binary senros."
+date: 2018-10-30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: minut.svg
+ha_category: BinarySensor
```
Comment (ludeeus) [inline source/_components/binary_sensor.point.markdown]:
```suggestion
ha_category: Binary Sensor
```

File: source/_components/sensor.point.markdown [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+- **temperature**: Temperature in °C.
+- **humidity**: Percentage of humidity in the air.
+- **pressure**: Pressure in hPa.
+- **sound_level**: Sound level in dBa.
+
+
+For installation instructions, see [the Point component](/components/point/).
+
+<p class='note'>
+The Point is just active occasionaly so the sensors are only updated every hour or so.
```
Comment (ludeeus) [inline source/_components/sensor.point.markdown]:
```suggestion
The Point is just active occasionally so the sensors are only updated every hour or so.
```

File: source/_components/sensor.point.markdown [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: minut.svg
+ha_category: Sensor
+ha_iot_class: "Cloud Polling"
+---
+
+Each Point exposes the following Sensors:
```
Comment (ludeeus) [inline source/_components/sensor.point.markdown]:
```suggestion
Each Point exposes the following sensors:
```
Unsure about this one(?)

**Review by ludeeus — COMMENTED** [review]
Added some suggestions for correcting some typos :)

**Comment by balloob** [comment]
This PR needs to include on every page the release it was included.

**Comment by fredrike** [comment]
> This PR needs to include on every page the release it was included.

What do you mean @balloob ?

File: source/_components/binary_sensor.point.markdown [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+title: "Minut Point Binary Sensor"
+description: "Access your Minut Point Events as binary sensors."
+date: 2018-10-30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: minut.svg
+ha_category: Binary Sensor
+ha_iot_class: "Cloud Push"
```
Comment (frenck) [inline source/_components/binary_sensor.point.markdown:13]: `ha_release` is missing.

File: source/_components/point.markdown [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+date: 2018-10-30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: minut.svg
+ha_category: Hub
+featured: true
+ha_iot_class: "Cloud Polling"
+---
```
Comment (frenck) [inline source/_components/point.markdown:16]: `ha_release` is missing.

File: source/_components/sensor.point.markdown [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+description: "Access your Minut Point Sensors."
+date: 2018-10-30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: minut.svg
+ha_category: Sensor
+ha_iot_class: "Cloud Polling"
+---
```
Comment (frenck) [inline source/_components/sensor.point.markdown:15]: `ha_release` is missing.

**Review by frenck — CHANGES_REQUESTED** [review]
@fredrike I've left you a review as comments on the places where the release information was missing, as stated by balloob.

**Comment by balloob** [comment]
It's `ha_release`

File: source/_components/point.markdown [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+logo: minut.svg
+ha_category: Hub
+featured: true
+ha_release: 0.83
+ha_iot_class: "Cloud Polling"
+---
+
+The Point component is the main component to integrate the [Minut Point](https://minut.se/). To connect Point, you will have to [sign up for a developer account](https://minut.com/community/developers/) and get a `client_id` and `client_secret` using the `callback url`  as `base_url` + `/api/minut` eg. `http://localhost:8123/api/minut`. The `client_id` and `client_secret` should be used as below. 
+
+Once Home Assistant is started, a configurator will pop up asking you to Authenticate your Point account via a link when you follow the link and have clicked on **Accept** you will be redirected to the `callback url` and the Point integration will be automatically configures and you can go back to the original dialog and press **Submit**.
```
Comment (frenck) [inline source/_components/point.markdown]:
```suggestion
Once Home Assistant is started, a configurator will pop up asking you to Authenticate your Point account via a link when you follow the link and have clicked on **Accept** you will be redirected to the `callback url` and the Point integration will be automatically configured and you can go back to the original dialog and press **Submit**.
```

File: source/_components/point.markdown [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+
+### {% linkable_title Configuration %}
+
+```yaml
+# Example configuration.yaml entry
+point:
+  client_id: CLIENT_ID
+  client_secret: CLIENT_SECRET
+```
+
```
Comment (frenck) [inline source/_components/point.markdown:30]: Configuration options list is missing.

Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

**Review by frenck — CHANGES_REQUESTED** [review]
@fredrike Thank you for adding the `ha_release` tags 👍 

I've now done a general review and left you some comments. Could you please take a look?
Thanks already! 👍

**Comment by fredrike** [comment]
@frenck Thank you for your comments, I've updated accordingly.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks for updating @fredrike! 👍 

I'll go ahead and merge it in.

**Pull request merged**